pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -369,6 +368,16 @@ func Test_server_Exists(t *testing.T) {
 		insertNum = 1000
 	)
 
+	jpStr := "こんにちは"
```

Comment:
I think utf8Str is better.
And I won't post a suggestion because it is better to use the bulk replace in the editor.